service-loadbalancer/service_loadbalancer.go

```diff
@@ -431,18 +484,33 @@ func dryRun(lbc *loadBalancerController) {
 }
 
 func main() {
+	clientConfig := kubectl_util.DefaultClientConfig(flags)
 	flags.Parse(os.Args)
 	cfg := parseCfg(*config)
 	if len(*tcpServices) == 0 {
 		glog.Infof("All tcp/https services will be ignored.")
 	}
-	go healthzServer()
-
-	proc.StartReaper()
 
 	var kubeClient *unversioned.Client
 	var err error
 
+	defErrorPage, err := newStaticPageHandler("/etc/haproxy/errors/404.http")
```

Comment:
optional nit: 
This is a little awkward because you're settings something in the struct, then you're loading a page and setting the same things outside. Would make threading it etc hard.  Since file:// is defined in the url spec http will handle it, eg (not sure if this code will compile):

```go
const defaultErrorPage = "file:///etc/haproxy/errors/404.http"

newStaticPageHandler(*errorPage, defaultErrorPage) *staticPageHandler {
  t := &http.Transport{}
  t.RegisterProtocol("file", http.NewFileTransport(http.Dir("/")))
  c := &http.Client{Transport: t}
  s := staticPageHandler{c}
  if err := s.loadUrl(*errorPage); err != nil {
    s.loadUrl(defaultErrorPage)
  }
}

func (s *staticPageHandler) loadErrorPage(url string) {
	res, err := s.c.Get(url)
	if err != nil {
		log.Fatalf("%v", err)
	}
	defer res.Body.Close()
	body, err := ioutil.ReadAll(res.Body)
	if err != nil {
		log.Fatalf("%v", err)
	}
	log.Printf("%v", string(body))
        // set page
}

newStaticPagehandler(*errorPage, defaultErrorPage)

```


However it's trivial enough that I would accept if you just add a TODO comment for now